enclave_manager/avalon_enclave_manager/singleton/singleton_enclave_manager.py

```diff
@@ -152,13 +157,13 @@ def _process_work_order_by_id(self, wo_id):
                 logger.error("Received empty work order corresponding " +
```

Comment:
Lot of functionalities in _process_work_order_by_id() is common for Singleton and WPE. Please move common functionalities to base class